userbenchmark/ddp_experiments/__init__.py

```diff
@@ -390,6 +390,8 @@ def get_backend_name(model_args):
                         breakname = "withbreaks" if has_breaks else "nobreaks"
                         if has_breaks:
                             copied_model_args.append("--optimize_dynamo_ddp")
+                        if "inductor" in backend_name:
+                            copied_model_args.extend(["--torchinductor_cudagraph", "False"])
```

Comment:
i worry about unaligning with inductor's defaults.  i think inductor is planning on ditching cudagraphs anyway, which would be good to confirm.  if checking this in temporarily helps you its fine with me - but otherwise i'm more interested in pushing this change upstream to inductor to force the issue

Reply:
@wconstab considering discussion from today's meeting (i.e. we expect to tell users to turn off cudagraphs if their model is big / OOMs, or turn it on for the perf improvements) is this still an issue?

Also what did you mean by upstreaming the change? Move it into DDPOptimizer? Move it into the main torchbench models? Or in dynamo in general?

Reply:
so, i think the jury is still out on what the best way to pipe this kind of config through our benchmark or user model code to inductor is.  But i'll stamp for now to unblock.